### ISSUE TITLE
Allow changing default click behavior for HTTP links to open in external browser

### ISSUE THREAD

**Issue by JakeSays** [wavebox/waveboxapp#1217]
I never use the built-in wavebox browser when viewing an embedded link in an email. The built-in browser doesn't have access to my saved passwords and auto fill data.

Can you please add the ability to change the default click behavior to open in an external browser? Wavebox is the only email client I've ever used that requires a control-click to open an embedded link.

**Comment by nmat** [comment]
In wavebox 10 you can change the setting for:

![image](https://user-images.githubusercontent.com/383031/76710299-a1c92480-6706-11ea-8338-217a39bc2639.png)

I tested to change the setting to be: Default Browser and it opens the link in the default browser.

**Comment by JakeSays** [comment]
@nmat Yeah I'm not using wavebox 10, nor do I plan to. Not a fan at all.
